keyboards/eyeohdesigns/sprh/keymaps/default/keymap.c

```diff
@@ -0,0 +1,47 @@
+/* Copyright 2019 IOD
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+#define ______ KC_TRNS
+#define QWERTY 0
+#define FUNCTN 1
```

Comment:
We heavily recommend using enums here, since they're easier to maintain long term. 
```suggestion
enum custom_layers {
    _QWERTY,
    _FUNCTN,
};
```